toqito/channel_props/tests/test_is_extremal.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for the is_extremal function in the toqito library."""
+
+import unittest
```

Comment:
@CreativeBinBag We use pytest not unittest. Take a look at the existing unit tests to help you.